scarb/tests/package.rs

```diff
@@ -1535,6 +1535,8 @@ fn package_with_package_script() {
     #[cfg(not(windows))]
     let script_code = indoc! { r#"
       touch text.txt
+      cd ../..
```

Comment:
why are you cd-ing? everything needed for building should be copied into package directory already 🤔

Reply:
Moved all included items into the `target/package`